README.md

```diff
@@ -181,6 +181,16 @@ This will remove any entries in the `context`, `session`, `cgi_data` and `params
 
 If `Honeybadger.Filter.Default` does not suit your needs, you can implement your own filter. See the `Honeybadger.Filter.Mixin` module doc for details on implementing your own filter.
 
+## Filtering arguments
+
+Honeybadger supports showing arguments in the stacktrace. This is made possible by setting `filter_args` to `false`
+
+```elixir
+config :honeybadger,
+filter_args: false
+```
+NB:The cases where arguments are available in the stacktrace are limited.This currently works for `FunctionClauseError`
```

Comment:
```suggestion
```

The cases where arguments are available in the stacktrace are limited. Typically, this only works for a `FunctionClauseError`.
```